bin/cloudgov/setup-egress-for-apps

```diff
@@ -160,6 +165,19 @@ for app in $applist; do
       fi
     fi
 
+    if [ "$app" = cron ]; then
```

Comment:
Just a heads-up, I'm giving some thought to splitting up this script (and setup-egress-for-space). Something that stands out to me here is that we're starting to overload it with, I guess, "things that should be enforced"?